examples/tinywl/StackWorkspace.qml

```diff
@@ -117,6 +127,11 @@ Item {
                 dockModel: dock.model
                 creator: toplevelComponent
             }
+
+            Component.onCompleted: {
+                x = Helper.leftExclusiveMargin + 10
```

Comment:
If the popup's parent is a layer surface, we shouldn't set offset to its position.

Reply:
popup support will in implemented in another pr.